operator_ui/src/selectors/jobsShowRunCount.ts

```diff
@@ -1,3 +1,3 @@
-export default state => {
-  return state.jobRuns.currentJobRunsCount
-}
+import { IState } from '../connectors/redux/reducers/index'
+
+export default ({ jobRuns }: IState) => jobRuns.currentJobRunsCount
```

Comment:
lets add return type too

Reply:
This doesn't need to return type as it's inferred